src/elements/quick-note/QuickNote.ts

```diff
@@ -394,6 +394,11 @@ export class QuickNoteElement extends OutsideClick implements OnInit, OnDestroy,
         let word = this.getWordAtCursor().trim();
         if (this.isTagging) {
             let symbol = this.config.triggers[this.taggingMode];
+            if (word.indexOf(symbol) === -1) {
+                this.zone.run(() => {
```

Comment:
I took out the zone.run and called hideResults directly and the component never actually gets destroyed